Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -189,6 +194,13 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
             make.bottom.equalTo(syncCardView)
         }
 
+        // We need a refrence to the sync pages textlabel so we can adjust the constraints
+        if let index = introViews.index(of: syncCardView), index < textLabels.count {
+            let syncTextLabel = textLabels[index]
+            syncTextLabel.snp.makeConstraints { make in
+                make.bottom.equalTo(signInButton.snp.top).offset(-IntroViewControllerUX.SyncButonTopPadding)
```

Comment:
I am surprised we didn't have this constraint before (I didn't read the rest of the layout code, I would have thought the bottom of that label would already be glued to the top of the view below it).

Reply:
The whole vc is pretty wonky. But it works!